docs/tutorial/part06.rst

```diff
@@ -2,5 +2,648 @@
+                'nodes_type': 'Address',
+                'nodes_related': self.serialize_relationships(self.addresses.all(), 'REGISTERED_ADDRESS'),
+            },
+            {
+                'nodes_type': 'Officer',
+                'nodes_related': self.serialized_realtionships_of_type('Officer'),
+            },
+        ]
+
+Officer::
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'Address',
+                'nodes_related': self.serialize_relationships(self.addresses.all(), 'REGISTERED_ADDRESS'),
+            },
+            {
+                'nodes_type': 'Entity',
+                'nodes_related': self.serialize_relationships(self.entities.all(), 'OFFICER_OF'),
+            },
+            {
+                'nodes_type': 'Officer',
+                'nodes_related': self.serialized_realtionships_of_type('Officer'),
+            },
+        ]
+
+Other::
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'Officer',
+                'nodes_related': self.serialized_realtionships_of_type('Officer'),
+            },
+            {
+                'nodes_type': 'Entity',
+                'nodes_related': self.serialized_realtionships_of_type('Entity'),
+            },
+            {
+                'nodes_type': 'Address',
+                'nodes_related': self.serialize_relationships(self.addresses.all(), 'REGISTERED_ADDRESS'),
+            },
+        ]
+
+
+Call the serialize relationships methods on the helpers
+=======================================================
+
+On your ``fetch_api/utils.py`` the ``fetch_node_details`` function change put this above the return
+statement::
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+So the function should be looking something like this::
+
+    def fetch_node_details(node_info):
+        node_type       = node_info['node_type']
+        node_id         = node_info['node_id']
+        node            = MODEL_ENTITIES[node_type].nodes.get(node_id=node_id)
+        node_details    = node.serialize
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+        return node_details
+
+Return the json to the frontend
+===============================
+
+Now if you call your functions ``fetch_node_details`` and ``fetch_nodes`` should be returning the
+same data but in a way that is JSON parsable, so let's change a couple things in order of returning
+this data that we need.
+
+In your project settings file for us it is ``paradise_papers_search/settings/base.py`` but if you

```

Comment:
They only have one setting file not two like the project does since they do not need the production settings file. Also, the code REST_FRAMEWORK is already there so we should talk if we are going to leave it there or add it here. 👍